src/Deck/Deck.js

```diff
@@ -14,21 +14,41 @@ class Deck extends Component {
   render() {
     return (
       <article className="deck--container">
-        <h2 className="deck--name">{this.props.userDeck.deckName}</h2>
-        <img
-          className="deck--image"
-          src="https://musingsofalifelongnerd.files.wordpress.com/2015/01/mtg-card-back.jpg"
-          alt={this.props.userDeck.deckName}
-        />
-        <button onClick={this.handleClick}>
-                {this.props.userDeck.cards.length} cards
-        </button>
-        <div className="price--container">
-          <h3 className="price--value">${this.props.userDeck.price}</h3>
-          <button onClick={this.addToFaveDecks}>
-            Add to Favorite Decks
-          </button>
-        </div>
+        {this.props.cardAreaView === 'compareDecks' && (
+          <div>
+            <h2 className="deck--name">{this.props.userDeck.deckName}</h2>
+            <img
+              className="deck--image"
+              src="https://musingsofalifelongnerd.files.wordpress.com/2015/01/mtg-card-back.jpg"
+              alt={this.props.userDeck.deckName}
+            />
+            <button onClick={this.handleClick}>
+                    {this.props.userDeck.cards.length} cards
+            </button>
+            <div className="price--container">
+              <h3 className="price--value">${this.props.userDeck.price}</h3>
+              <button onClick={this.addToFaveDecks}>
+                Add to Favorite Decks
+              </button>
+            </div>
+          </div>
+        )}
+        {this.props.cardAreaView === 'faveDecks' && (
+          <div>
+            <h2 className="deck--name">{this.props.userDeck.deckName}</h2>
+            <img
+              className="deck--image"
+              src="https://musingsofalifelongnerd.files.wordpress.com/2015/01/mtg-card-back.jpg"
+              alt={this.props.userDeck.deckName}
+            />
+            <button onClick={this.handleClick}>
+                    {this.props.userDeck.cards.length} cards
+            </button>
+            <div className="price--container">
+              <h3 className="price--value">${this.props.userDeck.price}</h3>
+            </div>
+          </div>
+        )}
```

Comment:
Good idea. I think the button is the only part that needs to be conditionally rendered, if the rest is going to be the same. i.e.
```
{this.props.cardAreaView === 'compareDecks' && (
<button onClick={this.addToFaveDecks}>
  Add to Favorite Decks
</button>
)}
```

Reply:
This might cause some conflicts with my pr. I did not realize you were going to be working in Deck. We can figure it out tomorrow when we meet back up!